pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: graphql_flutter
 description: A GraphQL client for Flutter, bringing all the features from a modern GraphQL client to one easy to use package.
-version: 1.0.0+4
+version: 0.0.0-development
```

Comment:
Why are we starting from 0.0.0?

Reply:
The version gets overwritten, by the deploy job. So to avoid having to commit the version via the CI, I removed it altogether and let the semver be leading.

Reply:
I think keeping the version number in sync would be a nice to have, so I made #250 to track it separately